src/lightning_app/cli/cmd_clusters.py

```diff
@@ -93,10 +93,14 @@ def create(
 
         click.echo(f"${resp.id} cluster is ${resp.status.phase}")
 
-    def list(self):
+    def get_clusters(self):
         resp = self.api_client.cluster_service_list_clusters(phase_not_in=[V1ClusterState.DELETED])
+        return resp.clusters
+
+    def list(self):
+        clusters = self.get_clusters()
         console = Console()
-        console.print(ClusterList(resp.clusters).as_table())
+        console.print(ClusterList(clusters).as_table())
```

Comment:
I'd either remove the `get_clusters` function and merge it into `list()` - if not used anywhere else. 
Or (probably better) make the `get_clusters` return `ClusterList` object - this would reduce passing between functions OpenAPI objects - and use our internal objects for representation.

Reply:
Done!